README.md

```diff
@@ -1035,28 +1077,30 @@ Most navigations are cross-document navigations. Same-document navigations can h
 
 Here's a summary table:
 
-|Trigger|Cross- vs. same-document|Fires `navigate`?|`event.userInitiated`|
-|-------|------------------------|-----------------|---------------------|
-|Browser UI (back/forward)|Either|No|—|
-|Browser UI (non-back/forward fragment change only)|Always same|Yes|Yes|
-|Browser UI (non-back/forward other)|Always cross|No|—|
-|`<a>`/`<area>`|Either|Yes|Yes|
-|`<form>`|Either|Yes|Yes|
-|`<meta http-equiv="refresh">`|Either|Yes|No|
-|`Refresh` header|Either|Yes|No|
-|`window.location`|Either|Yes|No|
-|`window.open(url, name)`|Either|Yes|No|
-|`history.{back,forward,go}()`|Either|Yes|No|
-|`history.{pushState,replaceState}()`|Always same|Yes|No|
-|`appHistory.{back,forward,navigateTo}()`|Either|Yes|No|
-|`appHistory.{push,update}()`|Either|Yes|No|
-|`document.open()`|Always same|Yes|No|
-
-Some notes:
-
-- Regarding the "No" values for the "Fires `navigate`?" column: recall that we need to disallow abusive pages from trapping the user by intercepting the back button. To get notified of such non-interceptable cases after the fact, you can use `currentchange`.
-
-- Today it is not possible to intercept cases where other frames or windows programatically navigate your frame, e.g. via `window.open(url, name)`, or `history.back()` happening in a subframe. So, firing the `navigate` event and allowing interception in such cases represents a new capability. We believe this is OK, but will report back after some implementation experience. See also [#32](https://github.com/WICG/app-history/issues/32).
+|Trigger|Cross- vs. same-document|Fires `navigate`?|`e.userInitiated`|`e.cancelable`|`e.canRespond`|
+|-------|------------------------|-----------------|-----------------|--------------|--------------|
+|Browser UI (back/forward)|Either|Yes|Yes|No|Yes †*|
+|Browser UI (non-back/forward fragment change only)|Always same|Yes|Yes|Yes|Yes *|
+|Browser UI (non-back/forward other)|Always cross|No|—|—|—|
+|`<a>`/`<area>`|Either|Yes|Yes ‡|Yes|Yes *|
+|`<form>`|Either|Yes|Yes ‡|Yes|Yes *|
+|`<meta http-equiv="refresh">`|Either|Yes|No|Yes|Yes *|
```

Comment:
Drive-by nitpick: meta Refresh and the Refresh header will always be cross-document, since they're effectively a reload. Same with form submission.

Reply:
Not true in Firefox, it turns out! https://boom-bath.glitch.me/meta-refresh.html fun times.